benchmark/metrics/test/test_cputime.py

```diff
@@ -0,0 +1,219 @@
+        """
+        d = get_node_cpu_times(
+            _LocalRunner(),
+            Node(uuid=uuid4(), public_address=IPAddress('10.0.0.1')),
+            ['n0n-exist'],
+        )
+
+        def check(result):
+            self.assertIs(result, None)
+
+        d.addCallback(check)
+
+        return d
+
+
+class ComputeChangesTests(SynchronousTestCase):
+    """
+    Test computation of CPU time change between two measurements.
+    """
+
+    def test_compute_change(self):
+        """
+        Process measurements are handled correctly.
+        """
+        labels = ['node1', 'node2']
+        before = [{'foo': 3, 'bar': 5}, {'foo': 10, 'bar': 2}]
+        after = [{'foo': 4, 'bar': 5}, {'foo': 12, 'bar': 5}]
+        result = compute_change(labels, before, after)
+        self.assertEqual(result, {
+            'node1': {'foo': 1, 'bar': 0},
+            'node2': {'foo': 2, 'bar': 3},
+            })
+
+    def test_compute_change_new_proc(self):
+        """
+        Process that only appears in ``after`` is ignored.
+        """
+        labels = ['node1']
+        before = [{'foo': 500}]
+        after = [{'foo': 555, 'bar': 5}]
+        result = compute_change(labels, before, after)
+        self.assertEqual(result, {'node1': {'foo': 55}})
+
+    def test_compute_change_lost_proc(self):
+        """
+        Process that only appears in ``before`` is ignored.
+        """
+        labels = ['node1']
+        before = [{'foo': 555, 'bar': 5}]
+        after = [{'foo': 600}]
+        result = compute_change(labels, before, after)
+        self.assertEqual(result, {'node1': {'foo': 45}})
+
+    def test_compute_change_error_before(self):
+        """
+        Error in ``before`` measurement provides ``None`` result.
+        """
+        labels = ['node1']
+        before = [None]
+        after = [{'foo': 555, 'bar': 5}]
+        result = compute_change(labels, before, after)
+        self.assertEqual(result, {'node1': None})
+
+    def test_compute_change_error_after(self):
+        """
+        Error in ``after`` measurement provides ``None`` result.
+        """
+        labels = ['node1']
+        before = [{'foo': 555, 'bar': 5}]
+        after = [None]
+        result = compute_change(labels, before, after)
+        self.assertEqual(result, {'node1': None})
+
+
+class CPUTimeTests(TestCase):
+    """
+    Test top-level CPU time metric.
+    """
+
+    def implements_IMetric(self):
+        """
+        CPUTime provides the IMetric interface.
+        """
+        verifyClass(IMetric, CPUTime)
+
+    def test_cpu_time(self):
+        """
+        Fake Flocker cluster gives expected results.
+        """
+        node1 = Node(uuid=uuid4(), public_address=IPAddress('10.0.0.1'))
+        node2 = Node(uuid=uuid4(), public_address=IPAddress('10.0.0.2'))
+        metric = CPUTime(
+            Clock(), FakeFlockerClient([node1, node2]),
+            _LocalRunner(), processes=['init'])
+        d = metric.measure(lambda: 1)
```

Comment:
Could you add a comment to explain that the function being passed here is essentially a no-op?